recipes/birch/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "birch" %}
+{% set version = "0.0.35" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c6d8176f012ba3518945c6a84bd7e3cf46899983f23e7e52911121930e40e768
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Are you sure that a compiler is required? This seems like pure python.

Reply:
You are correct, that shouldn't have been there.